### ISSUE TITLE
debug coffeescript?

### ISSUE THREAD

**Issue by jots** [Microsoft/vscode#5963]
- VSCode Version: 1.0.0
- OS Version: ubuntu 14.04

Steps to Reproduce:

1.  add `"sourceMaps": true,` to launch.json
2. set a breakpoint in the coffeescript file.

The breakpoint doesn't trigger.  Does debugging coffeescript work if you have the sourcemap in the same directory?  I specify program as: `"program": "${workspaceRoot}/app.js"` should it be something else?

**Comment by weinand** [comment]
@jots I just tried the simple coffee program from http://coffeescript.org and VS Code 1.1.1 on 'elementary' (a variant of ubuntu).

I saved the sample program as coffee.coffee and compiled with `coffee -c -m coffee.coffee'. Then I created this launch config:
```ts
		{
			"name": "Coffee",
			"type": "node",
			"request": "launch",
			"program": "${workspaceRoot}/coffee.coffee",
			"cwd": "${workspaceRoot}",
			"sourceMaps": true,
			"stopOnEntry": true,
			"smartStep": false
		}
```
This is the result:
![may-22-2016 14-20-40](https://cloud.githubusercontent.com/assets/1898161/15453925/bf1323b6-2028-11e6-9176-c7fdcae883f5.gif)

So this works for me.
